server/boot/challenge.js

```diff
@@ -233,7 +233,7 @@ module.exports = function(app) {
             req.flash('info', {
               msg: dedent`
                 Once you have completed all of our challenges, you should
-                join our <a href=\"//gitter.im/freecodecamp/HalfWayClub\"
+                join our <a href=\"https://gitter.im/freecodecamp/HalfWayClub\"
                 target=\"_blank\">Half Way Club</a> and start getting
```

Comment:
@ltegman The actually issue with this is the escaping of double quotes. Since this is the new template string syntax, the double quotes end up having two brackets, which destroys the link. The https can stay but the escapes need to be removed